liquibase-extension-testing/src/main/java/liquibase/extension/testing/testsystem/command/TestSystemUpCommand.java

```diff
@@ -0,0 +1,70 @@
+package liquibase.extension.testing.testsystem.command;
+
+import liquibase.Scope;
+import liquibase.command.AbstractCommandStep;
+import liquibase.command.CommandArgumentDefinition;
+import liquibase.command.CommandBuilder;
+import liquibase.command.CommandResultsBuilder;
+import liquibase.extension.testing.testsystem.TestSystem;
+import liquibase.extension.testing.testsystem.TestSystemFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestSystemUpCommand extends AbstractCommandStep {
+
+    public static final String[] COMMAND_NAME = {"sdk", "system", "up"};
+
+    public static final CommandArgumentDefinition<String> NAME;
+    public static final CommandArgumentDefinition<String> VERSION;
+    public static final CommandArgumentDefinition<String> PROFILES;
+    public static final CommandArgumentDefinition<Boolean> ACCEPT_LICENSES;
+
+    static {
+        CommandBuilder builder = new CommandBuilder(COMMAND_NAME);
+
+        NAME = builder.argument("name", String.class).required()
+                .description("The name of the system to").build();
+        VERSION = builder.argument("version", String.class)
+                .description("Override version to use").build();
+        PROFILES = builder.argument("profiles", String.class)
+                .description("Set profile(s)").build();
+        ACCEPT_LICENSES = builder.argument("acceptLicense", Boolean.class)
+                .description("Accept licenses for any systems used/accessed")
+                .defaultValue(false)
```

Comment:
Why is this a default of false? This seems like a nuisance.

Reply:
Yes, it is a nuisance, but I'm not a lawyer and in talking to @kevin-at-datical he figured that 3rd party license agreements should be made by the user, not automatically by us. So this is what I did to try to keep it as non-annoying as possible while still making the license decision be in the user's hands.

Besides this argument, there is also a liquibase.sdk.testSystem.acceptLicenses config which can be set in your liquibase.properties or liquibase.sdk.local.yaml file too.

Reply:
That makes sense. When documentation comes around, we should make sure that the config option is in the docs.